src/derivatives/introduction.md

```diff
@@ -15,15 +15,15 @@ Examples:
 A defaced T1w image would typically be made during the curation process and is thus under raw
 
 ```Text
-sourcedata/private/sub-01/anat/sub-01_T1w.nii.gz
+sourcedata/defaced/sub-01/anat/sub-01_T1w.nii.gz
```

Comment:
The idea here was that the undefaced image is in `sourcedata/private`, while the defaced image is in `sub-01/`.

Reply:
Sorry, my bad. I had meant to write `original` or `with_face` there, to contrast to the `defaced`. Or `to_be_kept_private` would also work. 

This change was not crucial to the PR, but I found `private` a bit unclear. Feel free to keep `private` if you prefer that.

Reply:
How about:

```suggestion
sourcedata/pre-deface/sub-01/anat/sub-01_T1w.nii.gz
```

Reply:
oh yes, that would be good